shell/platform/fuchsia/dart_runner/BUILD.gn

```diff
@@ -209,6 +210,7 @@ template("jit_runner_package") {
     binary = "dart_jit${product_suffix}_runner"
 
     cmx_file = rebase_path("meta/dart_jit${product_suffix}_runner.cmx")
+    cml_file = rebase_path("meta/dart_jit${product_suffix}_runner.cml")
```

Comment:
OOC I noticed we don't have a corresponding `cml_file = dart_aot${product_suffix}_runner.cml` line in this PR - is that an oversight or intentional?